lib/registrant.rb

```diff
@@ -0,0 +1,26 @@
+class Registrant
+  attr_reader :name,
+              :age,
+              :permit
+
+  attr_accessor :license_data
+
+  def initialize(name, age, permit = false)
+    @name = name
+    @age= age
+    @permit = permit
+    @license_data = {
+                    :written => false, 
+                    :license => false, 
+                    :renewed => false
+                  }
+  end
+
+  def permit?
+    @permit
+  end
+
+  def earn_permit
+    @permit = true
```

Comment:
You might make this more robust by adding an `if` statement that checks that they can earn a permit `if @age >= 15`